packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -517,6 +520,13 @@ abstract class FlutterCommand extends Command<void> {
     );
   }
 
+  void _printDeprecationWarning() {
+    if (deprecated) {
+      printStatus('$warningMark The "$name" command will be deprecated in a future version of Flutter.');
```

Comment:
... command is deprecated and will be removed in a ...

Reply:
I should have re-read this in the morning with fresh eyes.